boto3/resources/base.py

```diff
@@ -100,6 +105,8 @@ def __init__(self, *args, **kwargs):
                 continue
 
             if name not in self.meta.identifiers:
+                logger.debug('kwargs: %s', kwargs)
```

Comment:
Did you mean to leave these log statements in?  If so can we make the more complete, add more context, etc?